src/nineLessons/Switch.java

```diff
@@ -0,0 +1,31 @@
+package nineLessons;
+
+public class Switch {
+    public static void main(String[] args) {
+        String dayOfTheWeek="Wednesday";
+        switch (dayOfTheWeek) {
```

Comment:
use the newest way to define switch:

switch (dayOfTheWeek) {
case "Monday" -> System.out.println("The second working day");
......
default -> o.toString();
}